experimental/framework/presence/.eslintrc.cjs

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	extends: [require.resolve("@fluidframework/eslint-config-fluid/strict"), "prettier"],
+	parserOptions: {
+		project: ["./tsconfig.json", "./src/test/tsconfig.json"],
+	},
+	rules: {
```

Comment:
Consider starting with no exceptions or overrides, and only adding them as needed in their respective PRs.

Reply:
Thanks for the nudge - I had remove a questionable one. And revisiting I think some were really out of date. (pulled from prototype started so long ago.)